charts/stan/values.yaml

```diff
@@ -0,0 +1,146 @@
+##################################
+#                                #
+#  NATS Streaming Configuration  #
+#                                #
+##################################
+stan:
+  image: astronomerinc/ap-nats-streaming:0.18.0
+  pullPolicy: IfNotPresent
+  replicas: 3
+
+  # Cluster name, generated into a name from the
+  # release name by default.
+  clusterID:
+
+  logging:
+    debug: true
+    trace:
+
+  # In case of NATS not being embedded then
+  # an URL for NATS is required.
+  nats:
+    url: nats://astronomer-nats:4222
```

Comment:
@andriisoldatenko does this rely on the release name being `astronomer`? If so can we make it dynamic on the release name?

Reply:
is it possible to use template variables in values?

Reply:
@schnie done in 511804f149a929eb5b05fbb705ed866708ec95b5